log.capnp

```diff
@@ -1910,7 +1910,11 @@ struct Event {
     gpsNMEA @3 :GPSNMEAData;
     can @5 :List(CanData);
     controlsState @7 :ControlsState;
-    sensorEvents @11 :List(SensorEventData);
```

Comment:
in order to keep backwards compatibility with old logs, we don't remove fields, but add `DEPRECATED` at the end of the name and move it to the bottom of the struct with the other deprecated fields